src/NServiceBus.Core.Tests/API/APIApprovals.ApproveNServiceBus.approved.txt

```diff
@@ -2581,8 +2581,6 @@ namespace NServiceBus.Routing.MessageDrivenSubscriptions
     {
         public Publishers() { }
         public void Add(System.Type eventType, string publisher) { }
-        public void Add(System.Reflection.Assembly eventAssembly, string publisher) { }
```

Comment:
> sounds reasonable. Make sure you document that

I don't think we have reasonable documentation of the advanced API but I'll add it to the PoA to make sure it will be done.